honeybee_3dm/helper.py

```diff
@@ -26,3 +35,187 @@ def get_unit_system(file_3dm):
 
     return unit
 
+
+def child_layer_control(config, layer_name):
+    """Checks if child layers are requested for a layer in the config file."""
+
+    if 'include_child_layers' in config['layers'][layer_name] and \
+        config['layers'][layer_name]['include_child_layers']:
+        return True
+    else:
+        return False
+
+
+def grid_controls(config, layer_name):
+    """Returns grid controls for a layer from the config.
+
+    Args:
+        grid_controls: A list of grid controls from the config file
+
+    Returns:
+        A tuple of grid controls (grid_size, grid_offset) if valid 
+            grid settings are found in the config file or None
+    """
+
+    if 'grid_settings' in config['layers'][layer_name] and \
+            config['layers'][layer_name]['exclude_from_rad']:
+        grid_controls = config['layers'][layer_name]['grid_settings']
+        return grid_controls['grid_size'], grid_controls['grid_offset']
+    else:
+        return None
```

Comment:
You can remove these lines. Python by default returns None if there is no return.